src/test/java/ludogorie_soft/reservations_platform_api/controller/PropertyControllerTest.java

```diff
@@ -0,0 +1,158 @@
+package ludogorie_soft.reservations_platform_api.controller;
+
+import ludogorie_soft.reservations_platform_api.dto.PropertyRequestDto;
+import ludogorie_soft.reservations_platform_api.dto.PropertyResponseDto;
+import ludogorie_soft.reservations_platform_api.service.PropertyService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class PropertyControllerTest {
+
+    @Mock
+    private PropertyService propertyService;
+
+    @InjectMocks
+    private PropertyController propertyController;
+
+    @BeforeEach
```

Comment:
Better use @ExtendWith(MockitoExtension.class) because it validates framework usage and detects incorrect stubbing. More about this topic here (even though it is for JUnit4) https://stackoverflow.com/questions/10806345/runwithmockitojunitrunner-class-vs-mockitoannotations-initmocksthis

Reply:
@BeforeEach has been replaced with @ExtendWith